src/core/opamSHA.mli

```diff
@@ -18,17 +18,8 @@ val sha512_file: string -> string
 val hash_file: [< `SHA256 | `SHA512 ] -> string -> string
 
 
-val sha256_bytes: Bytes.t -> string
+val sha256_string: string -> string
```

Comment:
why renaming these functions from `sha256` to `sha256_string` ?

Reply:
It’s not a renaming, only removal of functions that are not used.